resources/dicts/events/death/mountainous/general.json

```diff
@@ -218,5 +220,4 @@
             "warrior"
         ]
         }
```

Comment:
Looks like you might be missing a } here!

Reply:
thanks! I was wondering what was going wrong